src/main/res/values/strings.xml

```diff
@@ -207,30 +207,30 @@
 
     <!-- Notifications -->
     <string name="sync_title">Sync process</string>
-    <string name="sync_inprogress">Sync in progress...</string>
-    <string name="fatal_error">Fatal error ! </string>
+    <string name="sync_inprogress">Sync in progress …</string>
```

Comment:
those ... are not supported on android if i remember

Reply:
Ellipses are UTF-8. Works just fine.

Reply:
Oh no, … is wrong?

Reply:
I wouldn't say wrong, just not how Android wants it to be expressed. See e.g. http://www.mysamplecode.com/2012/05/android-replace-with-ellipsis-character.html?m=1

Reply:
Well, that isn't in other Nextcloud Android resources, and it is very prone to errors in translation.